language/analyze_v2/analyze_entities_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package language_v2
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+)
+
+func TestAnalyzeEntities(t *testing.T) {
+	buf := new(bytes.Buffer)
+
+	text := "Google is located in Mountain View."
+	err := analyzeEntities(buf, text)
+	if err != nil {
+		t.Errorf("TestAnalyzeEntities: %v", err)
```

Comment:
nit: replace `t.Errorf()` with `t.Fatalf`. If the call to `analyzeEntities()` returns err, all the other tests will fail too.

Reply:
Done.